ocaml/xapi/xapi_pci_helpers.ml

```diff
@@ -28,8 +28,17 @@ type pci = {
   subsystem_vendor: pci_property option;
   subsystem_device: pci_property option;
   related: string list;
+  driver_name: string option;
 }
 
+let get_driver_name address =
+    try
+      let driver_path = Unix.readlink (Printf.sprintf "/sys/bus/pci/devices/%s/driver" address) in
+      match Astring.String.cut ~sep:"/" ~rev:true driver_path with
+      | Some (prefix, suffix) -> Some suffix
+      | None -> None
+    with _ -> None
```

Comment:
Please add a debug line printing the exception value in case of failure

Reply:
Sure, I will